### ISSUE TITLE
Make a unique url for each demonstration image in the app

### ISSUE THREAD

**Issue by delhomer** [Oslandia/mapillary_classification#78]
In order to reuse known images in the demo pages, one should consider unique URLs that correspond to every image in the web application.
*Exemple:*
```
APP_URL/predictor_demo/semantic_segmentation/shapes/shape_01234.png
```
will give the prediction result for image `shape_01234.png`.

For instance, the URL is the same for each image:

```
APP_URL/predictor_demo/semantic_segmentation/shapes
```

**Comment by delhomer** [comment]
Addressed by PR #85, in review.

**Comment by delhomer** [comment]
Solved by PR #85 (commit [46225f6](dbb84537b38ccc63b12058d3d2d1d6d06))
